src/main/java/fr/insee/survey/datacollectionmanagement/questioning/dto/SearchSurveyUnitDto.java

```diff
@@ -0,0 +1,8 @@
+package fr.insee.survey.datacollectionmanagement.questioning.dto;
+
+public interface SearchSurveyUnitDto {
+
+    String getIdSu();
+    String getIdentificationCode();
+    String getIdentificationName();
```

Comment:
why an interface for this ?

Reply:
Passing through an interface acts as a converter between my SearchSurveyUnitDto and the result of my query.